sys/include/net/nanocoap.h

```diff
@@ -242,6 +242,7 @@ typedef struct {
     const char *path;               /**< URI path of resource               */
     unsigned methods;               /**< OR'ed methods this resource allows */
     coap_handler_t handler;         /**< ptr to resource handler            */
+    void *context;                  /**< ptr to user defined context data   */
```

Comment:
I prefer naming this attribute 'data' rather than 'context'. To me, 'context' includes a connotation of an enveloping entity. This pointer is just user-defined data for the resource. I don't feel strongly about this though -- it's just my preference and I'm OK with 'context'.

Reply:
I chose context to be consistent with netdev...